google/cloud/spanner/samples/samples.cc

```diff
@@ -529,6 +529,36 @@ void AddColumn(std::vector<std::string> const& argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+void AddStoringIndex(std::vector<std::string> const& argv) {
+  if (argv.size() != 3) {
+    throw std::runtime_error(
+        "add-storing-index <project-id> <instance-id> <database-id>");
+  }
+  // [START spanner_create_storing_index]
+  using google::cloud::future;
+  using google::cloud::StatusOr;
+  [](std::string const& project_id, std::string const& instance_id,
+     std::string const& database_id) {
+    google::cloud::spanner::DatabaseAdminClient client;
+    google::cloud::spanner::Database database(project_id, instance_id,
+                                              database_id);
+    future<StatusOr<
+        google::spanner::admin::database::v1::UpdateDatabaseDdlMetadata>>
+        future = client.UpdateDatabase(database, {R"""(
+            CREATE INDEX AlbumsByAlbumTitle2 ON Albums(AlbumTitle)
+                STORING (MarketingBudget))"""});
+    StatusOr<google::spanner::admin::database::v1::UpdateDatabaseDdlMetadata>
+        result = future.get();
```

Comment:
Nit: I think it's nice to name these StatusOr variables after the type/object that they (might) contain. Like in this case, maybe name it `metadata`. That way, the rest of the code reads nicely as if the caller has a (smart)pointer to a metadata object, which they kinda do.

Reply:
Thanks, renamed the variable. I'll merge when green